src/main/java/frc/robot/Robot.java

```diff
@@ -48,11 +48,11 @@ public void robotPeriodic() {
   /** This function is called once each time the robot enters Disabled mode. */
   @Override
   public void disabledInit() {
+    m_robotContainer.resetToAbsolutePositions();
   }
 
   @Override
   public void disabledPeriodic() {
-    m_robotContainer.resetToAbsolutePositions();
```

Comment:
Do we want to undo this and remove elevator reset from restToAbsolutePositions?

Reply:
Yeah. I want to test this tomorrow before we merge because the wrist encoder was being kinda funky today and this might be why